### PR TITLE
Added Novacoin

### PR DESCRIPTION
**Pull request by svost**

### CODE REVIEW

File: lib/coins/nvc.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+var common = {
+  name: 'Novacoin',
+  unit: 'NVC'
+}
+
+var main = assign({}, {
+  port: 7777,
+  portRpc: 8344,
+  protocol: {
+    magic: 0xe4e8e9e5 // careful, sent over wire as little endian
```
Comment (fanatid) [inline lib/coins/nvc.js]: As I understand bytes should be reversed https://github.com/novacoin-project/novacoin/blob/db0f9d16fab4183f6156f2ac652526cf22d70dd7/src/main.cpp#L3128

File: lib/coins/nvc.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    public: 0x08,
+    scripthash: 0x14
+  }
+}, common)
+
+var test = assign({}, {
+  port: 17777,
+  portRpc: 18344,
+  protocol: {
+    magic: 0xcdf2c0ef
```
Comment (fanatid) [inline lib/coins/nvc.js]: same question as for mainnet

**Comment by fanatid** [comment]
I don't see `bip32` constants

**Comment by svost** [comment]
My fault (magic), bip32 - not applicable for NVC. I refresh pull request bit later.

**Comment by fanatid** [comment]
Novacoin doesn't has `bip32` constants? In any case you should put `bip32` with empty object, because otherwise code fails on `toBitcoinJS`/`toBitcore`

**Pull request closed without merging**